### PR TITLE
Add header, style and linter

### PR DESCRIPTION
**Pull request by Goruchie**
In this project, I added a "Hello Microverse!" header, a ".h1-style"(class) for the header, and added the Microverse linter folder.
PD: I also created a gitignore file to ignore test.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
```
Comment (mukhammadrizooff) [inline README.md]: - [ ] [REQUIRED]  Please add descriptive informations to `line 5` and `Author` section. And please implement social networks links.

**Review by mukhammadrizooff — APPROVED** [review]
Hi @Goruchie, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good implementations ✔️ 
- Descriptive commit history ✔️

## Required Changes ♻️
- [ ] [REQUIRED] Please [install linters]() correctly and full version because you install only custume mode of linters!

image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Goruchie) [reply README.md]: Hello @mukhammadrizooff , nice to meet you!
 Thank you for your fastness, I'll fix this the fastest I can, but first I have a doubt. The activities requirement says: "Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git". So I just copy-pasted the HTML and CSS folder to my project, the same as one of my partners did, but in his case, he was approved, so I'm a little bit confused right now.
 On the other hand, I'm going to fix the descriptive information that you told me to do, I really don't know how I forgot that, maybe I was just a little bit exhausted from today's work.
 So thank you very much and I'll wait for your answer to finish this activity.

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @Goruchie                :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making all the changes requested by the previous reviewers good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

- [ ] On your question above, I believe when you are required to do is, the files that should be ignored are the files in the node_modules folder. This is done easily as you have done adding the ```node_modules/``` in your .gitignore. Hope this answered you :smiley:  

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**